src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -446,6 +448,11 @@ public function load(array $configs, ContainerBuilder $container)
         $container->registerForAutoconfiguration(LoggerAwareInterface::class)
             ->addMethodCall('setLogger', [new Reference('logger')]);
 
+        $handlerAutoconfiguration = $container->registerForAutoconfiguration(ElasticsearchLogstashHandler::class);
```

Comment:
I would bind on HandlerInterface instead - all handlers that need an `http_client` cannot use the default one since it logs.